tools/external_ic.F90

```diff
@@ -253,13 +253,13 @@ subroutine get_cubed_sphere_terrain( Atm, fv_domain )
 
     tile_id = mpp_get_tile_id( fv_domain )
 
-       call get_tile_string(fname, 'INPUT/fv_core.res'//trim(gn)//'.tile', tile_id(n), '.nc' )
+    fname = 'INPUT/fv_core.res'//trim(gn)//'.nc'
```

Comment:
I *think* you can get rid of the '//trim(gn)//', this should already be added by fms2_io.

Reply:
I tested removing this and answers reproduce.